.travis.yml

```diff
@@ -1,31 +1,37 @@
 sudo: false
+
 language: node_js
 
 node_js:
-  - "6.10.2"
-  - "stable"
+  - 6.10.2
+  - node
```

Comment:
According to https://docs.travis-ci.com/user/languages/javascript-with-nodejs/, `node` is the "latest stable Node.js release". The same as `tip`, I believe, but it is explicitly *not* io.js.